src/main/js/page_objects/blueocean/bluePipelineRunDetail.js

```diff
@@ -110,6 +110,19 @@ module.exports.commands = [{
         // E.g. nothing on the pipeline graph that allows us to find it.
         return this;
     },
+    assertMultiBranchResult: function (callback) {
```

Comment:
don't think this is sufficiently re-usable to be parked in here. It means you have to jump all over the code to get the slightest hint of what the test actually does .